src/lib/cardUtilFunc.java

```diff
@@ -0,0 +1,45 @@
+package lib;
+
+public class cardUtilFunc {
+
+    // method to compare 2 cards
+    // card1 > card2 => 1   card2 > card1 => -1   card1=card2 => 0
+    public static int cardComparator(String card1, String card2, char trump) {
+
+        if(card1.equalsIgnoreCase(card2)){
+            return 0;
+        }
+
+        char suit1 = card1.charAt(1), suit2 = card2.charAt(1);
```

Comment:
Use utils in line 13

Reply:
which constant in util can be used in line 13 ?

Reply:
utils.suit_index